docs/src/examples/components/Provider/Types/ProviderExampleRendererFelaPluginFallbackValue.shorthand.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { Segment } from '@stardust-ui/react'
+
+const ProviderExampleShorthand = () => (
+  <Segment
+    content="Segment with blue border"
+    styles={{ border: ['1px solid red', '1px solid blue'] as any }}
```

Comment:
We should update the typings for this, currently an array is not a valid `border` value...

Reply:
If we want to use visual tests for this I would propose to add one red segment with `['1px solid red', '1px solid invalid']` to verify that both values from the array are written to the styles.